### PR TITLE
Add blocking getRange usage example

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**

### CODE REVIEW

**Comment by nemanja-boric-sociomantic** [comment]
Total diff is screwed, better review commit by commit.

File: src/dlsproto/client/UsageExamples.d [hunk 1]
```diff
@@ -256,6 +256,29 @@ unittest
...
+        protected override void example ( )
+        {
+            // Assigns the getRange request and provides opApply-like
+            // task blocking interface.
+
+            // buffer for the result
+            void[] buf;
+
+            foreach ( key, val; this.dls_client.blocking.getRange("channel", buf,
+                                         0x1234, 0x5678))
```
Comment (gavin-norman-sociomantic) [inline src/dlsproto/client/UsageExamples.d]: Seemingly random indentation...?

Comment (gavin-norman-sociomantic) [reply src/dlsproto/client/UsageExamples.d]: Maybe:
```D
foreach ( key, val;
    this.dls_client.blocking.getRange("channel", buf, 0x1234, 0x5678))
```
?

**Review by gavin-norman-sociomantic — APPROVED** [review]
One tiny comment on formatting.

File: src/dlsproto/client/UsageExamples.d (hunk 1 above)
Comment (nemanja-boric-sociomantic) [reply src/dlsproto/client/UsageExamples.d]: That looks better, thanks. I couldn't figure it out for some reason :(

**Comment by nemanja-boric-sociomantic** [comment]
Updated.

**Pull request merged**